subprojects/snapshots/snapshots.gradle.kts

```diff
@@ -25,6 +25,7 @@ description = "Tools to take immutable, comparable snapshots of files and other
 dependencies {
     implementation(library("guava"))
     implementation(library("jsr305"))
+    implementation(library("slf4j_api"))
```

Comment:
Not relevant here, but we really shouldn't need to declare SLF4J and JSR-305 everywhere we use them. We should make sure that all relevant projects get the dependency automatically.

Reply:
agree

Reply:
https://github.com/gradle/gradle-private/issues/2411